build/Targets/Imports.targets

```diff
@@ -520,4 +520,30 @@
     </ItemGroup>
   </Target>
 
+  <!-- 
+    Prepare inputs for Symbol Store publishing.
+     
+    The imported targets populate ArtifactsSymStoreDirectory with links to binaries and Windows PDBs produced either 
+    directly by the build or converted from Portable PDBs produced by the build.
+    In official build the content of ArtifactsSymStoreDirectory is uploaded to a symbol server.
+  -->
+  <PropertyGroup>
+    <UsingToolPdbConverter>true</UsingToolPdbConverter>
+    <PublishOutputToSymStore Condition="'$(Nonshipping)' == 'true'">false</PublishOutputToSymStore>
+    <ArtifactsSymStoreDirectory Condition="'$(DeveloperBuild)' != 'true' and '$(OS)' == 'Windows_NT'">$(BaseOutputPath)$(Configuration)\SymStore\</ArtifactsSymStoreDirectory>
+  </PropertyGroup>
```

Comment:
Can we merge all these properties in with the unconditioned property group highher up in the file? Trying to keep property groups together when possible so it's easier to grok the meaning of files.

Reply:
OK.

---
In reply to: [147460899](https://github.com/dotnet/roslyn/pull/22850#discussion_r147460899) [](ancestors = 147460899)